rdkshellimage.h

```diff
@@ -45,7 +46,9 @@ namespace RdkShell
             bool loadJpeg(std::string fileName, unsigned char *&image, int32_t &width, int32_t &height);
             bool loadPng(std::string fileName, unsigned char *&image, int32_t &width, int32_t &height);
             bool loadBmp(std::string fileName, unsigned char *&image, int32_t &width, int32_t &height);
+            bool loadAPng(std::string fileName, unsigned char *&image, int32_t &width, int32_t &height);
             bool loadPngFromData(const char* imageData, int32_t imageSize, unsigned char *&image, int32_t &width, int32_t &height);
+            void BlendOver(unsigned char **rows_destination, unsigned char **rows_source, unsigned int x, unsigned int y, unsigned int width, unsigned int height);
```

Comment:
Please ensure the method names are camel case starting with a lower case letter to keep consistency with the other RDKShell APIs.